common/src/stack/pylib/stack/gen.py

```diff
@@ -872,3 +872,129 @@ def generate_bash(self):
 		profile  = [ '#! /bin/bash' ]
 		profile.extend(self.shellSection.generate())
 		return profile
+
+	def generate_ansible(self):
+		playbook = []
+		playbook.extend(self.shellSection.generate())
+		return playbook
+
+
+class AnsibleProfileTraversor(MainTraversor):
+	def pre(self):
+		self.tasks = {
+			'install-post': [],
+			'boot-pre': [],
+			'boot-post': []
+		}
+
+	def traverse_stack_script(self, node):
+		stage  = self.getAttr(node, 'stack:stage',  default='install-post')
+		chroot = self.getAttr(node, 'stack:chroot', default='true')
+
+		# QUESTION: Do we want to ignore these stages in Ansible?
```

Comment:
I believe the answer is yes, and that we should move as much of them as possible into the other environments